### ISSUE TITLE
50X Timeout errors

### ISSUE THREAD

**Issue by MrAutismo** [Dineshkarthik/telegram_media_downloader#80]
**Describe the bug**

the `unknown_errors.txt` is getting spammed with 

`2021-03-24 21:24:42.584803	[-503 Timeout]	upload.GetFile`
And 
`2021-03-24 21:26:48.816757	[-500 No workers running]	upload.GetFile`

**To Reproduce**
Share the config: `Please don't share your api_hash & api_id`
```yaml
api_hash: XXXXX
api_id: XXXXX
chat_id: belledelphinecosplay
file_formats:
  audio:
  - all
  document:
  - all
  video:
  - all
ids_to_retry: []
last_read_message_id: 0
media_types:
- audio
- photo
- video
- document
- voice

```

**Python Version**
Python: 3.8.2

**OS:**
The OS and its version: [e.g. Ubuntu 20.04]
Win 10 with subsystem Ubuntu 20.04 (ubuntu started the script)

**Comment by Dineshkarthik** [comment]
@MrAutismo can you explain what is the `unknown_errors.txt`?
Also, the `503` timeout error might be related to network issues can you retry the script and check?

I tried the above config and it succeeded without any error. (No downloads as there is no message in that channel)

**Comment by MrAutismo** [comment]
I dont know what that file is, it just appeared.

Usually network shouldnt be a problem as i never had any issues.
I just started the script again and am getting the same errors.

Had it running over night, got 3Gb in videos, but no images (and there are alot)

**Comment by Dineshkarthik** [comment]
Restrictions are not explicitly declared so that should not affect the download, regarding the `FLOOT_WAIT` this will be fixed after upgrading the Pyrogram to 1.2 which is added to the issues #86
